src/main/java/com/bina/varsim/tools/evaluation/VCFcompare.java

```diff
@@ -969,105 +1042,134 @@ public void setBed_filename(String bed_filename) {
         }
     }
 
-    class dual_idx {
-        public int idx;
-        public int full_idx;
+    class DualIdx {
+        public int splitVariantIndex;
+        public int wholeVariantIndex;
 
-        dual_idx(int idx, int full_idx) {
-            this.idx = idx;
-            this.full_idx = full_idx;
+        /**
+         * constructor for this class given known splitVariantIndex and wholeVariantIndex
+         * @param splitVariantIndex
+         * @param wholeVariantIndex
+         */
+        DualIdx(int splitVariantIndex, int wholeVariantIndex) {
+            this.splitVariantIndex = splitVariantIndex;
+            this.wholeVariantIndex = wholeVariantIndex;
         }
 
-        dual_idx() {
-            idx = -1;
-            full_idx = -1;
+        /**
+         * initialize to -1 which means no valid variant available
+         */
+        DualIdx() {
+            splitVariantIndex = -1;
+            wholeVariantIndex = -1;
         }
 
         public boolean equals(Object obj) {
             if (obj == null)
                 return false;
             if (obj == this)
                 return true;
-            if (!(obj instanceof dual_idx))
+            if (!(obj instanceof DualIdx))
                 return false;
 
             // use EqualsBuilder?
-            dual_idx temp = (dual_idx) obj;
-            if (idx != temp.idx) {
+            DualIdx temp = (DualIdx) obj;
+            if (splitVariantIndex != temp.splitVariantIndex) {
                 return false;
             }
 
-            return full_idx == temp.full_idx;
+            return wholeVariantIndex == temp.wholeVariantIndex;
 
         }
 
         @Override
         public String toString() {
-            return "dual_idx{" +
-                    "idx=" + idx +
-                    ", full_idx=" + full_idx +
+            return "DualIdx{" +
+                    "splitVariantIndex=" + splitVariantIndex +
+                    ", wholeVariantIndex=" + wholeVariantIndex +
                     '}';
         }
     }
 
-    class result_comparator {
+    class ResultComparator {
 
-        chrSearchTree<ValueInterval1D<Variant>> _true_store; // true variants
-        double _overlap_ratio;
-        boolean _overlap_complex;
-        int _wiggle;
-        boolean _ignore_ins_len;
+        chrSearchTree<ValueInterval1D<Variant>> trueVariantIntervalTree; // true variants
+        double overlapRatio;
+        boolean overlapComplex;
+        int wiggle;
+        boolean ignoreInsertionLength;
 
         // Results to store
         // this stores the indexes of the true variants matched
-        ArrayList<dual_idx> matches_hom = new ArrayList<>();
-        ArrayList<ArrayList<dual_idx>> matches_het = new ArrayList<>(2); // matches either parent
+        ArrayList<DualIdx> homozygousMatches = new ArrayList<>();
```

Comment:
Can you check if it's okay to use `List<DualIdx>` as the type here? We should try to use the most general type possible given the operations we intend to perform.